### PR TITLE
chore: rebase release-candidate

### PR DESCRIPTION
**Pull request by haverchuck**
*Description of changes:*
Rebases release-candidate branch

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt [hunk 1]
```diff
@@ -387,37 +386,13 @@ public class AuthCognito : FlutterPlugin, ActivityAware, MethodCallHandler, Plug
...
-              // if signIn flag is false, we assume user is signed out so throw exception
-              } else {
-                errorHandler.handleAuthError(flutterResult, AuthException.SignedOutException())
-              }
-            }
-          }
-        },
-        { error -> errorHandler.handleAuthError(flutterResult, error) }
-      )
+      var user: AuthUser? = Amplify.Auth.currentUser;
```
Comment (dnys1) [inline packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt:389]: Is this overwriting the changes made in release-candidate?

Comment (haverchuck) [reply packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt:389]: Weird... this wasn't even marked as a conflict.

Comment (haverchuck) [reply packages/amplify_auth_cognito/android/src/main/kotlin/com/amazonaws/amplify/amplify_auth_cognito/AuthCognito.kt]: @dnys1 Yeah  - I have no idea why, none of these were marked as conflicts. I think I have this manually resolved now.

**Review by Jordan-Nelson — APPROVED** [review]

**Comment by dnys1** [comment]
LGTM. Would integrate the following changes:

- Fix for failing CI: https://github.com/aws-amplify/amplify-flutter/pull/887
- Release verification via unpub: https://github.com/aws-amplify/amplify-flutter/pull/876

**Review by dnys1 — APPROVED** [review]

**Pull request merged**